test/JDBC/expected/BABEL-CROSS-DB-vu-verify.out

```diff
@@ -793,6 +793,372 @@ int
 ~~END~~
 
 
+-- tsql
+-- Cross-db testcases for statements in batch/block
+USE master;
+GO
+
+-- Batches having system objects
+BEGIN; SELECT DISTINCT 1 FROM my_babel_cross_db_vu_prepare_db1.dbo.sysdatabases; BEGIN; SELECT DISTINCT 1 FROM babel_cross_db_vu_prepare_master_t1; END; END;
```

Comment:
1. Let's also add test where we call `sys` objects as well like db1.sys.databases etc.
2. Create temp tables after executing cross-db stmt as DDL permissions work differently.

Reply:
Also let's also include below cases:
1. tests for statements in batch inside a transaction.
2. some cases where login/user has some default db and schemas with it and it's executing one cross-db query with other queries in a batch.

Reply:
> * Let's also add test where we call `sys` objects as well like db1.sys.databases etc.
> * Create temp tables after executing cross-db stmt as DDL permissions work differently.

Done.

> Also let's include below cases:
> 
> 1. tests for statements in batch inside a transaction.
> 2. some cases where login/user has some default db and schemas with it and it's executing one cross-db query with other queries in a batch.

Regarding, 2nd point - not adding anything for default db setting since it doesn't affect anything while query execution. It is only utilised while setting up a new db session.

Reply:
regarding default_schema, It is being tracked by BABEL-5050, needs separate fix.